packages/target-tools/delivery-api-client/models/Geo.d.ts

```diff
@@ -1,6 +1,9 @@
 export interface Geo {
     latitude?: number;
     longitude?: number;
+    countryCode?: string;
+    stateCode?: string;
+    city?: string;
```

Comment:
@jasonwaters Yes please I'd like your help. I'm curious how this works because https://git.corp.adobe.com/TnT/openapi-specs/pull/38 is not merged yet.